pkg/cmd/kafka/create/create.go

```diff
@@ -809,12 +833,48 @@ func determineFlowFromQuota(opts *options) error {
 	case opts.hasLegacyQuota:
 		opts.useLegacyFlow = true
 		return nil
+	case opts.useTrialFlow:
+		opts.useLegacyFlow = true
+		return nil
 	default:
 		opts.useEnterpriseFlow = true
 		return nil
 	}
 }
 
+func doesUserHaveTrialInstances(opts *options) (bool, error) {
+	conn, err := opts.f.Connection()
+	if err != nil {
+		return false, err
+	}
+
+	api := conn.API()
+
+	cfg, err := opts.f.Config.Load()
+	if err != nil {
+		return false, err
+	}
+
+	userName, ok := token.GetUsername(cfg.AccessToken)
+	if !ok {
+		return false, opts.f.Localizer.MustLocalizeError("kafka.create.error.cannotFindUserDetails")
+	}
+
+	list, _, err := api.KafkaMgmt().GetKafkas(opts.f.Context).Page(strconv.Itoa(1)).Size(strconv.Itoa(99)).Execute()
+	if err != nil {
+		return false, err
+	}
+
+	for i := int32(0); i < list.GetSize(); i++ {
+		kafka := list.Items[i]
+		if kafka.GetOwner() == userName && kafka.GetInstanceTypeName() == TrialInstanceType {
```

Comment:
ideally the API should allow to filter per instance_type. So that the call in line 863 is reduced to only filtering the developer instances that are in a org.
Can you open a JIRA about this and ping @pbraun